src/Model.js

```diff
@@ -184,6 +183,59 @@ const Model = class Model {
     }
 
     /**
+     * Update many-many relations for model.
+     * @param relations
+     */
+    _refreshMany2Many(relations) {
+        const ThisModel = this.getClass();
+        const This = this;
```

Comment:
well, I replaced, but some tests not passing after that - visually getting same result as expected, but because of **static property** it's not same by value. So need to patch tests for it. For now I would left it as is.